Source/URLDispatcher.spoon/init.lua

```diff
@@ -50,10 +52,20 @@ obj.url_redir_decoders = { }
 --- URL dispatch rules.
 ---
 --- Notes:
----  * A table containing a list of dispatch rules. Each rule should be its own table in the format: `{ "url pattern", "application bundle ID", "function" }`, and they are evaluated in the order they are declared.
+
```

Comment:
A blank line in the middle of docstrings is going to break the docs generation

Reply:
Thanks - fixed.